src/groups/rotation_action.jl

```diff
@@ -129,3 +129,60 @@ end
 function inverse_apply(A::RotationAroundAxisAction, θ, p)
     return apply(A, -θ, p)
 end
+
+###
+
+@doc raw"""
+    MatrixColumnwiseMultiplicationAction(
+        M::AbstractManifold,
+        On::GeneralUnitaryMultiplicationGroup,
+        AD::ActionDirection = LeftAction(),
+    )
+
+Space of actions of the (special) unitary or orthogonal group [`GeneralUnitaryMultiplicationGroup`](@ref)
```

Comment:
While checking for the name – the type is called action, but the documentation says its the space of the action., I think it is the second? Also the docs could include the subtype info (in line140)

Reply:
So, I'm not sure -- do you prefer the "action" or "space of actions" name?

I'll include the subtype info and move the constructor signature below.

Reply:
I just felt that, when the type is called action, the docs should not say its the space of actions?

Reply:
OK, I see, I'll remove "Space of" then.